README.md

```diff
@@ -415,6 +415,8 @@ This Four Keys project defines the key metrics as follows:
 
 *   The number of failures per the number of deployments. For example, if there are four deployments in a day and one causes a failure, that is a 25% change failure rate.
 
+For more information on the calculation of the metrics, see the [METRICS.md](METRICS.md)
+
 ## Color coding
 
 The dashboard has color coding to show the performance of each metric. Green is strong performance, yellow is moderate performance, and red is poor performance. Below is the description of the data that corresponds to the color for each metric.
```

Comment:
Why do we use three buckets here, when the DORA reports (for the last couple years) have four buckets?

Reply:
I lumped high and elite together.  Should I separate them back out?  They'd both be green.

Reply:
IMHO, yes? (Note that the quick check uses two different shades of green.) But let's talk it through. I'll open an issue for discussion.

Reply:
tracked in: https://github.com/GoogleCloudPlatform/fourkeys/issues/62